main.py

```
@@ -5,7 +5,8 @@
 module_path = Path(__file__).parent
 sys.path.append(str(module_path))
 
-from fastapi import FastAPI
+from typing import Dict, Any
```

Ruff (0.11.9)

Module level import not at top of file

(E402)